GitHubActions/GitHubActions.psd1

```diff
@@ -54,6 +59,10 @@
         ,'Write-ActionError'
         ,'Write-ActionInfo'
         ,'Write-ActionWarning'
+        ## Context-related
+        ,'Get-ActionContextMap'
+        ,'Get-ActionIssue'
+        ,'Get-ActionRepo'
```

Comment:
the `,` in here isn't needed at all, the newline is enough of a separator.

Reply:
I agree, but I was reviewing some other _official_ modules from MSFT, and I noticed they tend to separate the names in the manifest with commas even when they're not needed.  I wasn't sure if this was some style policy, so I just kinda went with it for now.  Now the reason I put it at the start rather than the end is because it allows you to move the entries around without having to worry about differences with the first/last entry.